_api-reference/snapshots/create-repository.md

```diff
@@ -43,6 +43,7 @@ Request field | Description
 `compress` | Whether to compress metadata files. This setting does not affect data files, which might already be compressed, depending on your index settings. Default is `false`. Optional.
 `max_restore_bytes_per_sec` | The maximum rate at which snapshots restore. Default is 40 MB per second (`40m`). Optional.
 `max_snapshot_bytes_per_sec` | The maximum rate at which snapshots take. Default is 40 MB per second (`40m`). Optional.
+`remote_store_index_shallow_copy` | Boolean | Whether or not the current snapshot is a full copy or shallow copy. 
```

Comment:
Customer needs to set this flag during create repository which when enabled, we will take snapshots of all the remote store enabled indices as shallow copy. maybe rephrase it to something like `Whether or not the snapshot of remote store enabled indices be captured as shallow copy.'